### ISSUE TITLE
Can not use Azure Blockchain: Generate Smart Contract UI

### ISSUE THREAD

**Issue by only4sim** [microsoft/vscode-azure-blockchain-ethereum#4]
I can find "Azure Blockchain: Generate Smart Contract UI" in Command Palette. However, when I want to use it,  it shows there are no matching commands.

**Comment by caleteeter** [comment]
Hi @only4sim weird, this basically means the commands for the extension didn't load fully.  Might be a timing issue.  In this preview release, the load times are not super optimized.  Few things to check.

* Close all instances of VS Code
* When you open VS Code, if the extension is installed, click on the Azure Blockchain tiles in the lower left corner.
![image](https://user-images.githubusercontent.com/8452367/57313303-580eb700-70bd-11e9-8477-fd53ff8c19fa.png)
* You should see a progress bar atop the Open Editor in the top left.  This should resolve (stop) after a few seconds.  Might vary on your machine.  When its fully loaded you should see Connect to consortium and Create Azure Blockchain Service in the lower right.
![image](https://user-images.githubusercontent.com/8452367/57313436-986e3500-70bd-11e9-8589-c8ee023206ca.png)

Do you see this?

**Comment by only4sim** [comment]
Yes, it's wired. I can see Connect to consortium and Create Azure Blockchain Service in the lower right. But still cannot find "Azure Blockchain: Generate Smart Contract UI". The specific situation is as shown below.

![Q1](https://user-images.githubusercontent.com/35210356/57343216-00338700-7175-11e9-89b0-dc44e37beb15.png)
![Q2](https://user-images.githubusercontent.com/35210356/57343218-00338700-7175-11e9-949e-7f96549145c0.png)

And I am using Win10. @caleteeter
